config/routes.rb

```diff
@@ -38,10 +38,13 @@
     end
     resources :parking
 
-    resources :users
     scope :users do
       post "upload", to: "users#upload"
+      post "create", to: "users#create"
+      patch "update/:id", to: "users#update"
     end
```

Comment:
This may be another/separate task, but I believe there should also be the option to delete users

Reply:
I thought about it. But deleting a user deletes their reservations, and that could be useful for analytics.